app/views/inquiry/feedbacks/index.html.erb

```diff
@@ -7,15 +7,15 @@
   end
 %>
 <% @tap_menu = proc do |item| %>
-  <%= link_to :show, { action: :show, id: CGI.escape(item.id) }.merge(s: params[:s]) %>
+  <%= link_to t('ss.links.show'), { action: :show, id: CGI.escape(item.id) }.merge(s: params[:s]) %>
```

Comment:
__[brakeman]__
* Unsafe parameter value in link_to href [:ref](http://brakemanscanner.org/docs/warning_types/link_to_href)

<!-- Comment by [SideCI](https://sideci.com) -->